notification-service-sdk/.openapi-generator/VERSION

```diff
@@ -1 +1 @@
-5.4.0
\ No newline at end of file
+6.2.0
```

Comment:
this change makes it inconsistent with workflow-service-sdk. Please update both

Reply:
Moved this change to different PR
Please see PR #289